### PR TITLE
Image support

### PR DESCRIPTION
**Pull request by kristoffer-zliide**
The latest versions of Lottie supports animations with images by packaging them with the animation in a zip file.

This PR enables image support in animations by updating to the latest version of Lottie.

### CODE REVIEW

File: android/src/main/java/de/simolus3/fluttie/FluttiePlugin.java [hunk 1]
```diff
@@ -227,23 +230,24 @@ public void onCompositionLoaded(@Nullable LottieComposition composition) {
...
-				try {
-					stream = assets.open(key);
-				} catch (IOException e) {
-					e.printStackTrace();
-				}
-				// Lottie will close the input stream for us
-				LottieComposition.Factory.fromInputStream(stream, listener);
+				LottieCompositionFactory.fromAsset(registrar.context(), key)
+						.addListener(listener)
+						.addFailureListener(failure);
```
Comment (simolus3) [inline android/src/main/java/de/simolus3/fluttie/FluttiePlugin.java:250]: Nice :+1:

File: android/src/main/java/de/simolus3/fluttie/FluttieAnimation.java [hunk 2]
```diff
@@ -39,6 +39,7 @@ public FluttieAnimation(FluttiePlugin plugin, TextureRegistry.SurfaceTextureEntr
 				(int) (scale * bounds.width()), (int) (scale * bounds.height()));
 
 		drawable = new LottieDrawable();
+		drawable.setCallback(plugin.getAnimationCallback());
```
Comment (simolus3) [inline android/src/main/java/de/simolus3/fluttie/FluttieAnimation.java:42]: Can you briefly help me understand why this `callback` is necessary now? Is it due to some changes in the updated Lottie version or should it have always been like this? Thanks!

**Review by simolus3 — COMMENTED** [review]
Thank you very much for this, I really appreciate this :+1: Please help me on that one question about the `callback`, then I'll merge this and create a new version on pub.

File: android/src/main/java/de/simolus3/fluttie/FluttieAnimation.java (hunk 2 above)
Comment (kristoffer-zliide) [reply android/src/main/java/de/simolus3/fluttie/FluttieAnimation.java:42]: I think it should have always been like that.

If you look through https://github.com/airbnb/lottie-android/blob/master/lottie/src/main/java/com/airbnb/lottie/LottieDrawable.java, there's a lot of usages of getCallback() that will just be NOPs if this call is not made.

Personally, I wasted quite some time on this little silent failure: https://github.com/airbnb/lottie-android/blob/master/lottie/src/main/java/com/airbnb/lottie/LottieDrawable.java#L846. Had that if statement not been there, at least this warning would have been written: https://github.com/airbnb/lottie-android/blob/master/lottie/src/main/java/com/airbnb/lottie/manager/ImageAssetManager.java#L39.

**Review by simolus3 — APPROVED** [review]
Alright, thanks for the detailed explanation! I'll merge this now & then upload a new version on pub later.

**Pull request merged**